CONTRIBUTING.md

```diff
@@ -23,6 +28,49 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Instruction notebook content
+
+### Learning Objectives
+
+Each notebook should begin with a "Learning objectives" section.
+This section contains a compact summary of the goals for the notebook, in the form of a bulleted list of objectives, introduced by the phrase:
+
+> This notebook will demonstrate how to
```

Comment:
> I don't love my introductory phrase. I'm open to suggestions.

I am not wedded to this idea and am outside of my wheelhouse, but reading the Carpentries resource... is the following more what we are looking for?

```suggestion
> When we complete this notebook, you will be able to:
```

Reply:
Yeah, I thought about something like this, I really don't like telling people what they will and won't be able to do. If they can't do it at the end, I want to imply that it is my failure, not theirs.

It is useful for planning, but not necessarily what I want to convey.

Reply:
Right, I don't disagree with you. The current introductory phrase (which I would be okay with!) doesn't leave room for certain action verbs like "describe" in my opinion. We may be fine with this.